### PR TITLE
adding cric updater and two reporters.

### PR DESCRIPTION
**Pull request by ivukotic**

### CODE REVIEW

File: atlas-xcache/Dockerfile [hunk 1]
```diff
@@ -12,6 +12,8 @@ RUN yum install -y atlas-xcache --enablerepo="$BASE_YUM_REPO" --enablerepo=osg-c
     yum clean all --enablerepo=* && rm -rf /var/cache/
 
 COPY update-agis-status.sh /usr/local/sbin/
+COPY update-cric-status.sh /usr/local/sbin/
+COPY reporter.py stats.py /
```
Comment (brianhlin) [inline atlas-xcache/Dockerfile:16]: Do these actually have to live in `/`?

**Comment by ivukotic** [comment]
Hi,

No. Where do you prefer them?

Ilija

**Comment by brianhlin** [comment]
`/usr/local/sbin` seems like a good place

**Comment by ddavila0** [comment]
How is this code supposed to be triggered?
Should we care about the curl return code in `update-cric-status.sh` ?

**Comment by ivukotic** [comment]
Hi,

It is triggered from preExec in my k8s deployments.
Adding logic to handle potential issues in connecting to CRIC is on my TODO list, but can’t do it right away.

Cheers,
Ilija

**Comment by ddavila0** [comment]
Do you really need the "insecure" (-k) option in curl ?

**Comment by ivukotic** [comment]
For now yes. Once they fix https I’ll remove it.

**Pull request merged**